dev/com.ibm.ws.context_fat/publish/files/internalFeatures/contextinternals-1.0.mf

```diff
@@ -0,0 +1,8 @@
+Subsystem-ManifestVersion: 1
```

Comment:
Lets add the following header to this manifest to make it clear that this is a test feature:
`IBM-Test-Feature: true`
We use this header in other test features as well

Reply:
done